components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Storage/PublishedNodesJobConverter.cs

```diff
@@ -71,130 +71,177 @@ public class PublishedNodesJobConverter {
+                            DataSet = new PublishedDataSetModel {
+                                DataSetSource = dataSetSource.Clone(),
+                            },
+                            DataSetFieldContentMask =
+                                    DataSetFieldContentMask.StatusCode |
+                                    DataSetFieldContentMask.SourceTimestamp |
+                                    (legacyCliModel.FullFeaturedMessage ? DataSetFieldContentMask.ServerTimestamp : 0) |
+                                    DataSetFieldContentMask.NodeId |
+                                    DataSetFieldContentMask.DisplayName |
+                                    DataSetFieldContentMask.ApplicationUri |
+                                    (legacyCliModel.FullFeaturedMessage ? DataSetFieldContentMask.EndpointUrl : 0) |
+                                    (legacyCliModel.FullFeaturedMessage ? DataSetFieldContentMask.ExtensionFields : 0),
+                            MessageSettings = new DataSetWriterMessageSettingsModel() {
+                                DataSetMessageContentMask =
+                                        (legacyCliModel.FullFeaturedMessage ? DataSetContentMask.Timestamp : 0) |
+                                        DataSetContentMask.MetaDataVersion |
+                                        DataSetContentMask.DataSetWriterId |
+                                        DataSetContentMask.MajorVersion |
+                                        DataSetContentMask.MinorVersion |
+                                        (legacyCliModel.FullFeaturedMessage ? DataSetContentMask.SequenceNumber : 0)
                             }
+                        }).ToList(),
+                        MessageSettings = new WriterGroupMessageSettingsModel() {
+                            NetworkMessageContentMask =
+                                    NetworkMessageContentMask.PublisherId |
+                                    NetworkMessageContentMask.WriterGroupId |
+                                    NetworkMessageContentMask.NetworkMessageNumber |
+                                    NetworkMessageContentMask.SequenceNumber |
+                                    NetworkMessageContentMask.PayloadHeader |
+                                    NetworkMessageContentMask.Timestamp |
+                                    NetworkMessageContentMask.DataSetClassId |
+                                    NetworkMessageContentMask.NetworkMessageHeader |
+                                    NetworkMessageContentMask.DataSetMessageHeader
                         }
-                    }).ToList();
+                    }
+                });
+                _logger.Information("Created {jobs} jobs in {elapsed}", result.Count(), sw.Elapsed);
+
+                var counter = 0;
+                foreach (var job in result) {
+                    _logger.Verbose("groupId: {group}", job.WriterGroup.WriterGroupId);
```

Comment:
@mregen, can you please add some more information to the log message. It is not clear how this log can be used. The same comment for the log bellow on line `215`.

Reply:
what kind of information do you mean? The output is pretty obvious:
```
[11:22:55 INF Microsoft.Azure.IIoT.OpcUa.Edge.Publisher.Models.PublishedNodesJobConverter] Read 1 items from published nodes file in "00:00:00.3329831" 
[11:22:55 INF Microsoft.Azure.IIoT.OpcUa.Edge.Publisher.Models.PublishedNodesJobConverter] Created 1 groups in "00:00:00.0024429" 
[11:22:55 INF Microsoft.Azure.IIoT.OpcUa.Edge.Publisher.Models.PublishedNodesJobConverter] Created 1 flattened groups in "00:00:00.1736669" 
[11:22:55 INF Microsoft.Azure.IIoT.OpcUa.Edge.Publisher.Models.PublishedNodesJobConverter] Created 1 jobs in "00:00:00.0672013" 
[11:22:55 INF Microsoft.Azure.IIoT.OpcUa.Edge.Publisher.Models.PublishedNodesJobConverter] Total count of OpcNodes after job conversion: 125012 
[11:22:55 INF Microsoft.Azure.IIoT.OpcUa.Edge.Publisher.Models.PublishedNodesJobConverter] Converted items to jobs in "00:00:00.3296201" 
```